app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/TestActivity.java

```diff
@@ -0,0 +1,171 @@
+package edu.grinnell.appdev.grinnelldirectory.Activity;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.grinnell.appdev.grinnelldirectory.APICaller;
+import edu.grinnell.appdev.grinnelldirectory.Interfaces.APICallerInterface;
+import edu.grinnell.appdev.grinnelldirectory.Model.Person;
+import edu.grinnell.appdev.grinnelldirectory.R;
+import edu.grinnell.appdev.grinnelldirectory.Model.User;
+
+/**
+ * Created by nicholasroberson on 2/22/17.
+ */
+
+public class TestActivity extends AppCompatActivity implements APICallerInterface {
```

Comment:
Excellent! I'm glad you tested this! We should move all future tests to the test package. We shouldn't have the Tests as part of our app's source code.Also, you should use a Unit Test. Please see this: https://developer.android.com/training/testing/unit-testing/local-unit-tests.html